src/hooks/use-agency-info.tsx

```diff
@@ -0,0 +1,24 @@
+// import { useStaticQuery, graphql } from 'gatsby';
```

Comment:
this is a custom hook as way to place all hooks into separate files. However, after moving everything to gatsby-node, this became moot. However I wanted to keep it in case I need to use it in the future. 

There is this repeated thing that occurs, I figure something out, then don't need but might need it in the future. So I keep it commented. There has to be a better way to save stuff so that I don't have to figure it out again.